Update webhook docs (#5994)
PostHog/posthog.com: contents/docs/webhooks/microsoft-teams.md

showTitle: true
---

> You can receive messages in Teams whenever any of your specified [actions](/docs/user-guides/actions) are triggered. 

Reviewer: ```suggestion
You can receive messages in Teams whenever an [action](/docs/user-guides/actions) triggers.
```

## How to setup the Teams webhook

### 1. Create an incoming webhook in Teams

1. Navigate to the channel where you want to add the webhook and select (•••) More Options from the top navigation bar.
1. Choose Connectors from the drop-down menu and search for Incoming Webhook.
1. Select the Configure button, provide a name, and, optionally, upload an image avatar for your webhook.
1. The dialog window will present a unique URL for that channel. Make sure that you copy and save the URL – we will need it in the next step.
1. Select the Done button. The webhook will now be available in the team channel.

For more details, see the [Microsoft Teams documentation](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook#add-an-incoming-webhook-to-a-teams-channel).

### 2. Setup the webhook in PostHog

Navigate to project settings in PostHog paste the url you copied from Teams into the "Webhook integration section".

![Add webhook integration](../../images/docs/webhooks/webhook-integration.png)

Click "Test & Save" and you should receive a message on Teams. 

### 3. Post actions to the webhook

> For each action that should be posted to Slack, select "Post to webhook when this action is triggered". You can also modify the [message format](/docs/webhooks#message-formatting).

Reviewer: ```suggestion
For each action that should be posted to Teams, select "Post to webhook when this action is triggered". You can also modify the [message format](/docs/webhooks#message-formatting).
```
